plugins/auto-sizes/includes/improve-calculate-sizes.php

```diff
@@ -360,3 +380,19 @@ function auto_sizes_filter_render_block_context( array $context, array $block, ?
 	}
 	return $context;
 }
+
+/**
+ * Retrieves the featured image attachment ID for a given post ID.
+ *
+ * @since 1.4.0
```

Comment:
Why `1.4.0`? Isn't this a new function?
```suggestion
 * @since n.e.x.t
```